lib/main.dart

```diff
@@ -35,6 +37,8 @@ class MyApp extends StatelessWidget {
         Routes.home: (context) => HomePage(),
         Routes.ppe: (context) => PPEView(),
         Routes.ppeOnGuidance: (context) => PPEOnGuidance(),
+        Routes.ppeOffGuidance1: (context) => PPEOffGuidance1(),
```

Comment:
could we please make this a bit more descriptive, so instead of `1` `method_1` for both the route name and the Widget names

Reply:
Cleaned up in 4d9ce74 and 18ebb49